mono/mini/mini-runtime.c

```diff
@@ -2981,9 +2986,13 @@ MONO_SIG_HANDLER_FUNC (, mono_sigint_signal_handler)
 	MonoException *exc;
 	MONO_SIG_HANDLER_GET_CONTEXT;
 
+	MONO_ENTER_GC_UNSAFE_UNBALANCED;
```

Comment:
This conflicts with https://github.com/mono/mono/pull/4132/files#diff-307195a8afdfeac6e3df25eef5aaf9d6R2865